src/main/kotlin/com/arabyte/arabyteapi/domain/auth/service/AuthService.kt

```diff
@@ -0,0 +1,84 @@
+package com.arabyte.arabyteapi.domain.auth.service
+
+import com.arabyte.arabyteapi.domain.auth.api.KakaoUserApi
+import com.arabyte.arabyteapi.domain.auth.dto.AuthorizeResponse
+import com.arabyte.arabyteapi.domain.auth.dto.LoginResponse
+import com.arabyte.arabyteapi.domain.auth.dto.RegisterRequest
+import com.arabyte.arabyteapi.domain.auth.dto.RegisterResponse
+import com.arabyte.arabyteapi.domain.auth.dto.jwt.RefreshAccessTokenRequestBody
+import com.arabyte.arabyteapi.domain.auth.dto.retrofit.KakaoUserResponse
+import com.arabyte.arabyteapi.domain.auth.util.JwtProvider
+import com.arabyte.arabyteapi.domain.user.entity.User
+import com.arabyte.arabyteapi.domain.user.service.UserService
+import com.arabyte.arabyteapi.global.enums.CustomError
+import com.arabyte.arabyteapi.global.exception.CustomException
+import jakarta.transaction.Transactional
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+
+@Service
+class AuthService(
+    @Value("\${spring.security.oauth2.client.registration.kakao.client-id}")
+    private val clientId: String,
+    private val userService: UserService,
+    private val kakaoUserApi: KakaoUserApi,
+    private val jwtProvider: JwtProvider
+) {
+    fun getKakaoUserInfo(accessToken: String): KakaoUserResponse {
```

Comment:
```suggestion
    private fun getKakaoUserInfo(accessToken: String): KakaoUserResponse {
```

밖으로 보여지지 않아도 되는 함수는 private으로 선언해주세요

Reply:
반영했습니다. 감사합니다.